icmd/internal/stub/main.go

```diff
@@ -1,3 +1,4 @@
+// Package main produces a test-binary used in tests.
```

Comment:
FWIW, I was a bit on the fence wether to disable this specific check in the linter or to add the (quite "boiler-plate-y" comments), but most packages already had a Package description, so I thought it'd be useful to be a bit more "strict", and run the linter.